pliers/extractors/bert_extract_features.py

```diff
@@ -0,0 +1,544 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Extract pre-computed feature vectors from BERT."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import codecs
+import collections
+import json
+import re
+
+from pliers.extractors import bert_modeling
+from pliers.extractors import bert_tokenization
+import tensorflow as tf
+
+import numpy as np
+
+flags = tf.flags
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string("input_file", None, "")
```

Comment:
I am concerned about implementing extractor specific / case specific CLIs within pliers. If we do, they should at least not be in extractors, but I have a feeling that these don't belong in pliers at all. This sort of code can certainly wrap pliers and live elsewhere, or maybe we could generalize this code and not make it specific to text embeddings (and work for all of pliers).